File: src/main/java/com/hank/guess/GameFor.java
package com.hank.guess;

import java.util.Scanner;

public class GameFor {

    public static void main(String[] args) {
        for (int i = 1; i <= 3; i++) {
            System.out.println(i);
        }
        Scanner scanner = new Scanner(System.in);
        for (int i = 1; i <= 5; i++) {
            int number = scanner.nextInt();
            System.out.println("第" + i + "次 : " + number);

        }

    }

}

File: src/main/kotlin/com/hank/GameFor.kt
package com.hank

import java.util.Random
import java.util.Scanner


fun main() {
    val secret = Random().nextInt(10) + 1
    println("secret = $secret")
    val scanner = Scanner(System.`in`)
    for (i in 1..4) {
        print("Please enter a number (${i}/4)")
        var number = scanner.nextInt()
        println("第${i}次$number")
        if (number > secret) {
            println("Lower")
        } else if (number < secret) {
            println("Higher")
        } else {
            println("Great, the number is $secret")
            break
        }

    }

}












